detox/local-cli/detox-test.js

```diff
@@ -68,25 +72,27 @@ function runMocha() {
   const debugSynchronization = program.debugSynchronization ? `--debug-synchronization ${program.debugSynchronization}` : '';
   const command = `node_modules/.bin/mocha ${testFolder} ${configFile} ${configuration} ${loglevel} ${cleanup} ${reuse} ${debugSynchronization} ${platform} ${artifactsLocation}`;
 
-  console.log(command);
   cp.execSync(command, {stdio: 'inherit'});
 }
 
 function runJest() {
+  const currentConfiguration = config.configurations && config.configurations[program.configuration];
+  const maxTestWorkers = _.get(currentConfiguration, 'maxTestWorkers', 1);
```

Comment:
Please use the same way to get configuration on all variables, either move all to ` _.get`, use `getConfigFor`, or "vanilla"